source/includes/_overview.md

```diff
@@ -2,18 +2,27 @@
 
 Welcome to the Hyperledger Iroha API docs!
 
-<iframe width="560" height="315" style="padding-left: 28px" src="https://www.youtube.com/embed/Hf_5j0b6TZM" frameborder="0" allowfullscreen></iframe>
+The documentation describes the endpoints for users to send transactions to peer network, containing one or many commands to perform allowed actions in the system, and also make queries to know the current state. Users are authenticated by their keypair and account id passed as the parameter with their transactions or queries.
 
-This API allows users to send transactions to peer network, containing one or many commands to perform allowed actions in the system, and also make queries to know current state.
-Users are authenticated by their keypair and account id, passed as the parameter with their transactions or queries.
+The API is organized around protobuf format, as Iroha is using gRPC in transport level. Iroha CLI and block store use JSON format to provide developer-friendly experience for contributors. Feel free to check both sections for protobuf and JSON while exploring docs.
 
-The API is organized around protobuf format, as Iroha is using gRPC on transport level. Iroha CLI and blockstore works with JSON format to provide developer-friendly experience for community members.  
+## Elevator pitch for Iroha
 
-To try out a basic API request, do the following:
+The system is a distributed ledger platform, capable of storing all history of state-changing actions (called transactions) in immutable linear history (called blockchain). The system is intended to be used in enterprise or between untrusted agents as **private chain platform**, but not as a public chain — key difference is that not everyone is allowed to store the whole history. A subset of users can only query some data (perform a query) if their accounts have required permissions, while others maintain a peer (Iroha is a peer network), which stores history. Reliability and security are provided with cryptographic capabilities and **Byzantine Fault Tolerant** consensus algorithm. The system is targeted to environments, where client applications are very diverse (desktop, various mobile platforms), and peer hardware varies from embedded systems to enterprise-class servers.
```

Comment:
I'm not sure but I think it's article lost before "private chain platform" words.
Also I didn't get the point of "The system is targeted to environments" sentence.

Reply:
IMO this paragraph should be right after welcome sentence (line 3).

Reply:
Okay, I will swap them

Reply:
@luckychess about env -- the sentence is about where the system is used or applied, and the range of application is very broad and diverse. This tells about portability and compatibility of the system